solutions/cron/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "repository": "https://github.com/vercel/examples.git",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@vercel/analytics": "^0.1.10",
+    "@vercel/edge-config": "^0.1.1",
+    "@vercel/examples-ui": "^1.0.4",
+    "ms": "^2.1.3",
+    "next": "canary",
```

Comment:
Do we need canary or just latest?

Reply:
that's what we're using for other examples too so I just copied and pasted